lib/glific/clients/common_webhook.ex

```diff
@@ -95,6 +96,26 @@ defmodule Glific.Clients.CommonWebhook do
     |> GoogleASR.speech_to_text(fields["results"], contact.language.locale)
   end
 
+  # This webhook will call Bhasini speech-to-text API
+  def webhook("speech_to_text_with_bhasini", fields) do
+    contact_id = Glific.parse_maybe_integer!(fields["contact"]["id"])
+    contact = get_contact_language(contact_id)
+    speech = fields["speech"]
+    user_id = fields["userID"]
+    ulca_apikey = fields["ulcaApiKey"]
+    pipeline_id = fields["pipelineId"]
+    base_url = fields["base_url"]
+
+    Bhasini.with_config_request(
+      speech,
+      user_id,
+      ulca_apikey,
+      pipeline_id,
```

Comment:
instead of this send the fields map and use that in the `with_config_request` where it is needed

Reply:
Done